CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.0.0 - 2022-10-30
+### Changed
+- Raised minimum required PHP version from `7.4` to `8.1`.
+- Upgraded lcobucci/jwt library from `3.3` to `4.2.1`
```

Comment:
If the update to 4.2.1 still have issues you can try rolling it back to the version 3.2.5 as it was the latest 3* version that works on both php 7+ and 8+. I found out the hard way. :)

Had similar issues in the past with the principals portal:
https://github.com/oat-sa/pdf-generator/pull/265